### DISCUSSION TITLE
Barbie og Ken - 2023/6/4

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#31] [vefsida-ummæli]
# Barbie og Ken - 2023/6/4

Strákarnir vildu ótrúlega mikið sjá kvikmyndina, Barbie.

https://www.biskupsreglan.com/frettir/barbie-og-ken

**Comment by TJ478** [comment]
Þetta var svo ótrúlega skemmtileg mynd.
